openedx/features/job_board/views.py

```diff
@@ -15,6 +16,21 @@ class JobListView(ListView):
     template_engine = 'mako'
 
 
-def show_job_detail(request):
-
-    return render_to_response('features/job_board/job_detail.html', {})
+def show_job_detail(request, job_id):
```

Comment:
let's use class based detailview of django to keep the harmony in the code as done in joblistview
`https://docs.djangoproject.com/en/3.0/ref/class-based-views/generic-display/`